fluss-lakehouse/fluss-lakehouse-paimon/src/main/java/com/alibaba/fluss/lakehouse/paimon/FlussLakehousePaimon.java

```diff
@@ -94,7 +98,7 @@ public static void main(String[] args) throws Exception {
                 new PaimonDataBaseSyncSinkBuilder(paimonConfig, flussConfig).withInput(input);
         paimonDataBaseSyncSinkBuilder.build();
 
-        System.out.println("Starting data tiering service to Paimon.....");
+        logger.info("Starting data tiering service to Paimon...");
```

Comment:
I'm not sure about this. I though it is intended to use standard output to print on the terminal? @luoyuxia